tests/test_eval_detection.py

```diff
@@ -34,41 +34,62 @@ def metrics(evaluator: DetectionEvaluator) -> DataFrame:
+    gts = DataFrame([{"width": 10, "height": 10, "length": 10}])
+    assert (dist_fn(dts, gts, DistFnType.SCALE) == 1 - 0.125).all()
 
 
 def test_orientation_distance() -> None:
-    """Test that the orientation distance function works."""
+    """Intialize a detection and a ground truth label with only orientation
+    parameters. Verify that calculated orientation error matches the expected
+    smallest angle between the detection and ground truth label.
+    """
     # check all of the 45 degree angles
     vecs_45_apart = [angle * np.array([0, 0, 1]) for angle in np.arange(0, 2 * np.pi, np.pi / 4)]
     for i in range(len(vecs_45_apart) - 1):
-        df1 = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_45_apart[i]).as_quat())}])
-        df2 = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_45_apart[i + 1]).as_quat())}])
-        assert np.isclose(dist_fn(df1, df2, DistFnType.ORIENTATION), np.pi / 4)
-        assert np.isclose(dist_fn(df2, df1, DistFnType.ORIENTATION), np.pi / 4)
+        dts = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_45_apart[i]).as_quat())}])
+        gts = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_45_apart[i + 1]).as_quat())}])
+        assert np.isclose(dist_fn(dts, gts, DistFnType.ORIENTATION), np.pi / 4)
+        assert np.isclose(dist_fn(gts, dts, DistFnType.ORIENTATION), np.pi / 4)
     # check all of the 90 degree angles
     vecs_90_apart = [angle * np.array([0, 0, 1]) for angle in np.arange(0, 2 * np.pi, np.pi / 2)]
     for i in range(len(vecs_90_apart) - 1):
-        df1 = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_90_apart[i]).as_quat())}])
-        df2 = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_90_apart[i + 1]).as_quat())}])
-        assert np.isclose(dist_fn(df1, df2, DistFnType.ORIENTATION), np.pi / 2)
-        assert np.isclose(dist_fn(df2, df1, DistFnType.ORIENTATION), np.pi / 2)
+        dts = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_90_apart[i]).as_quat())}])
+        gts = DataFrame([{"quaternion": quat_scipy2argo_vectorized(R.from_rotvec(vecs_90_apart[i + 1]).as_quat())}])
+        assert np.isclose(dist_fn(dts, gts, DistFnType.ORIENTATION), np.pi / 2)
+        assert np.isclose(dist_fn(gts, dts, DistFnType.ORIENTATION), np.pi / 2)
+
+
+def test_iou_aligned_3d() -> None:
+    """Intialize a detection and a ground truth label with only shape
+    parameters (only shape parameters due to alignment assumption).
+    Verify that calculated intersection-over-union matches the expected
+    value between the detection and ground truth label.
+    """
+    dt_dims = DataFrame([{"width": 10, "height": 3, "length": 4}])
+    gt_dims = DataFrame([{"width": 5, "height": 2, "length": 9}])
+
+    assert (iou_aligned_3d(dt_dims, gt_dims) == (40.0 / 270.0)).all()
```

Comment:
test looks great. thanks for adding this. One thing -- I would write the assert line as
```python
# intersection is 40 = 4*5*2 (min of all dimensions)
# union is the sum of the two volumes, minus intersection: 270 = 10*3*4 + 5*2*9 - 40
assert (iou_aligned_3d(dt_dims, gt_dims) == ( 40 / 270.0)).all()
```